test/unit/specs/components/staking/ModalWithdrawAllRewards.spec.js

```diff
@@ -0,0 +1,66 @@
+"use strict"
+
+import { shallowMount } from "@vue/test-utils"
+import ModalWithdrawAllRewards from "staking/ModalWithdrawAllRewards"
+
+describe(`ModalWithdrawAllRewards`, () => {
+  let wrapper, $store
+
+  beforeEach(() => {
+    $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      getters: {
+        bondDenom: `stake`,
+        distribution: { totalRewards: { stake: 10000000 } }
+      }
+    }
+    wrapper = shallowMount(ModalWithdrawAllRewards, {
+      mocks: {
+        $store
+      },
+    })
+  })
+
+  it(`shows withdaw rewards modal`, () => {
```

Comment:
how do you guys feel about the convention of starting the `it` blocks with `should`?

```suggestion
`it(`should show the withdraw rewards modal`, () => {`
```

Reply:
I feel we shouldn't block a PR because this small convention. Let's discuss it on other opportunity